src/sentry/mediators/sentry_app_installations/notifier.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+from sentry.http import safe_urlopen, safe_urlread
+from sentry.mediators import Mediator, Param
+from sentry.utils.cache import memoize
+
+
+class Notifier(Mediator):
+    install = Param('sentry.models.SentryAppInstallation')
+
+    def call(self):
+        self._send_webhook()
```

Comment:
Is there value in having another function at this stage? :bike::house:

Reply:
Probably not right now. I wrote it this way mostly to be consistent with the other Mediators. From a readability point of view, I like this, sort of, summary view of logic held within. Granted, it's one step here so :man_shrugging: